### DISCUSSION TITLE
Is it possible to extend the 300 characters limit for the server rules?

### DISCUSSION THREAD

**Discussion by nicfab** [mastodon/mastodon#18566] [Q&A]
I am trying to input some contents into the box server rules, but I receive the message "is too long (maximum is 300 characters)".

Are there any settings to allow extending the 300 characters limit?

**Comment by Gargron** [comment]
Even 300 characters is too long for how this feature is intended to be used and displayed. Example:

![C3AD8948-3465-4649-A86A-35255B9A8AE0](https://user-images.githubusercontent.com/184731/171035267-8ec12cc7-2a99-422d-ad03-6b7587b1147f.png)

Now imagine one of these bullet points would be 300 characters. First of all, the rules would no longer fit on the screen and require scrolling. The user would also no longer be able to skim them easily. This is specifically intended to be readable unlike generic terms of service which are pages long.

**Reply by nicfab** [reply]
Thank you
